x-pack/plugins/index_lifecycle_management/__jest__/client_integration/edit_policy/reactive_form/reactive_form.test.ts

```diff
@@ -0,0 +1,143 @@
+/*
```

Comment:
I like the direction this is heading in, it is an important category of form states to capture in tests since as something distinct from serialising or validation.

I guess I would just ask how this file will scale if new functionality or new tests for old functionality is added? Do we append to this file?

One way that I was thinking about this is as "workflows", but I have since come think of it as "features" of the UI. The UI has a feature for viewing JSON in a flyout, which is one thing, it also has a feature for helping users configure rollover.

Anyway, just throwing these out there! Let me know what you think!

Reply:
I think 'features' is a good system to try and separate tests further. My plan is to address the second large `edit_policy.test.ts` file in a follow up PR and probably introduce those categories into `reactive_form` folder.